### ISSUE TITLE
Automatically detect seed import path

### ISSUE THREAD

**Issue by imaginaryusername** [Electron-Cash/Electron-Cash#1155]
Feature request: 

When importing seeds, Electron-cash can potentially do an "auto-detect" for user friendliness. Check the common paths BIP39 0'/0', 145'/0', electrum-specific 145'/0' etc for transactions on the first five addresses, and import if one hits any transaction. 

The current seed path (electrum-specific 0'/145') can continue to be offered as part of the settings - perhaps one can make it a radio button to choose "specify Electron-cash seed bath" above "specify BIP39 path".

**Comment by cculianu** [comment]
Yes.  This is a very good suggestion. I want to do this soon -- it's not even very hard to do and would add a ton of usability to the app.  Sorry it took me so long to respond.  Very good idea.  I want to do this soon -- hopefully before release.  I would be very convenient for users if our software did the intelligent thing and offered them the obvious path.  I do think keeping an advanced option there so that advanced users can specify what they want is good, though.  👍

**Comment by acidsploit** [comment]
Additional, save the BIP39 seed and derivation path, so the user still has the option to do a backup (again) at a later time.
